quickstarts/nextcloud/config.yml

```diff
@@ -3,21 +3,31 @@ slug: nextcloud
 
 description: |
   ## Comprehensive monitoring for your Nextcloud server
-  You can use New Relic's infrastructure agent to monitor the performance of your Nextcloud server in near real-time, correlate transactions as they move across the app environment, and identify issues to troubleshoot.
-  ### Why monitor your Nextcloud server?
-  Use the power of our infrastructure monitoring agent and the Prometheus open metric integration to control, monitor, and secure the data on your Nextcloud server. Use our dashboards to get a high-level overview of your server's performance. 
-  - See a map of your server's processes to help you track down the source of your issues fast.
-  - Use our tools to help you maintain quick throughput and low latency on your server.
-  - Check the health of your database, application, and server Nextcloud dependencies.
+  Empower yourself with New Relic's cutting-edge Nextcloud monitoring system and gain complete control over the health of your platform. Stay ahead of the curve by monitoring your Nextcloud in real-time, connecting the dots between transactions across your application environment, and swiftly troubleshoot any issues that arise with ease  ### Why monitor your Nextcloud server?
+ 
+   ## Why monitor Nextcloud?
+ 
+  ### Solve login problems
+  Users may experience issues with logging into their Nextcloud accounts, such as incorrect login credentials or connection errors. 
+
+  ### Quickly identify performance issues
+  Nextcloud can become slow or unresponsive, especially if the server is running low on resources or if the network connection is slow. 
+
+  ### Identify storage limitations
+  Nextcloud can experience storage limitations if the server's disk space is running low, or if there are issues with the underlying storage infrastructure. 
+
+  ### Catch file syncing issues
+  Users may experience issues with file syncing, such as conflict errors or delayed updates, which can result in data loss or corruption. 
+
 
   ### What’s included?
   Use New Relic to monitor and observe your Nextcloud server to:
   - Be alerted on things like excessive heap memory usage and scrape errors.
   - Use dashboards to monitor crucial performance metrics, such as uptime, free space, total active users, and more.
   - Enable your Nextcloud admins to keep track of the status and operation of your server installations.
+  
 summary: |
-  The controllable self-hosted productivity platform. Monitor the server, check the condition of it and keep a watch on what happens to your files with our Infrastructure agent and Prometheus open metric integration.
-icon: logo.png
```

Comment:
Looks like we got rid of the icon field here!

Reply:
Added it back!